### PR TITLE
docs: sync Array/at

### PR DESCRIPTION
**Pull request by sedlukha**
### Description

Updates for Array/at method docs and some cleanups

### CODE REVIEW

File: files/ru/web/javascript/reference/global_objects/array/at/index.md [hunk 1]
```diff
@@ -36,6 +32,17 @@ at(index)
 
 Элемент массива, соответствующий переданному индексу. Если переданный индекс не может быть найден, возвращает {{jsxref('undefined')}}.
 
+## Описание
+
+Метод `at()` является эквивиалентом получения элементов массива с помощью квадратных скобок с использованием неотрицательного индекса. Например, `array[0]` и `array.at(0)` оба вернут первый элемент. Однако, при вычислении значения с конца массива, нельзя использовать `array[-1]` как в Python или R, потому что все значения внутри квадратных скобок трактуются буквально как строковые свойства. Из-за этого в конечном итоге будет прочитано как `array["-1"]`, что является нормальным строковым значением, а не индексом массива.
```
Comment (Saionaro) [inline files/ru/web/javascript/reference/global_objects/array/at/index.md:37]:
```suggestion
Метод `at()` является эквивиалентом получения элементов массива с помощью квадратных скобок с использованием неотрицательного индекса. Например, `array[0]` и `array.at(0)` оба вернут первый элемент. Однако, при вычислении значения с конца массива, нельзя использовать `array[-1]` как в Python или R, потому что все значения внутри квадратных скобок трактуются буквально как строковые свойства. Из-за этого попытка обращения к -1 элементу будет прочитана как `array["-1"]`, что является нормальным строковым значением, а не индексом массива.
```

File: files/ru/web/javascript/reference/global_objects/array/at/index.md [hunk 2]
```diff
@@ -36,6 +32,17 @@ at(index)
...
+
+Обычной практикой является получении числа элементов массива {{jsxref("Array/length", "length")}} и последующее вычисление значения индекса — например, `array[array.length - 1]`. Метод `at()` разрешает относительную индексацию, поэтому может быть сокращено до `array.at(-1)`. Формально, если `index < 0`, то используется `index + array.length`.
```
Comment (Saionaro) [inline files/ru/web/javascript/reference/global_objects/array/at/index.md:39]:
```suggestion
Обычной практикой является получении числа элементов массива {{jsxref("Array/length", "length")}} и последующее вычисление значения индекса — например, `array[array.length - 1]`. Метод `at()` разрешает относительную индексацию, поэтому может быть сокращено до `array.at(-1)`. Формально, если `index < 0`, то используется `index + array.length`.
```
```suggestion
Обычной практикой является получении числа элементов массива {{jsxref("Array/length", "length")}} и последующее вычисление значения индекса — например, `array[array.length - 1]`. Метод `at()` разрешает относительную индексацию, поэтому может быть сокращено до `array.at(-1)`.
```

File: files/ru/web/javascript/reference/global_objects/array/at/index.md [hunk 3]
```diff
@@ -82,6 +89,19 @@ const atWay = colors.at(-2);
 console.log(atWay); // Выведет: 'green'
 ```
 
+### Вызов at() в массивоподобных объектах
+
+Метод `at()` считывает свойство `length` для значения `this` и вычисляет запрашиваемый индекс.
```
Comment (Saionaro) [inline files/ru/web/javascript/reference/global_objects/array/at/index.md:94]:
```suggestion
Метод `at()` считывает свойство `length` для значения `this` и вычисляет индекс для обращения.
```

**Review by saionaro — COMMENTED** [review]
@sedlukha thank you for contributing to MDN!
I applied few small fixes: we're good to go now

**Pull request merged**